educe/rst_dt/corpus.py

```diff
@@ -36,9 +35,20 @@ class Reader(educe.corpus.Reader):
     def __init__(self, corpusdir):
         educe.corpus.Reader.__init__(self, corpusdir)
 
-    def files(self):
+    def files(self, exclude_file_docs=False):
+        """
+        Parameters
+        ----------
+        exclude_file_docs : boolean, optional (default=False)
+            If True, fileX documents are ignored. The figures reported by
+            (Li et al., 2014) on the RST-DT corpus indicate they exclude
+            fileN files, whereas Joty seems to include them.
+            fileN documents are more damaged than wsj_XX documents, e.g.
+            text mismatches with the corresponding document in the PTB.
+        """
```

Comment:
(now merged) I'm not entirely sure I would have put this option here; might have pushed it out to the surface layer (the extractor) and maybe wrapped around is_interesting instead.  But I think that's just me wanting a uniform interface for educe.corpus.